### DISCUSSION TITLE
Request - second yogh variant

### DISCUSSION THREAD

**Discussion by TheKnightWho** [psb1558/Junicode-font#125] [Ideas]
Hi (and sorry for yet another request!) - I noticed your new comments in the Record interpreter doc regarding yogh and how it often resembles an ezh, along with your solution of substituting the glyph with cv63.

Appendix D to Statutes of the Realm appears to show an insular g glyph playing a similar role, which they've dubbed to be the "Saxon character" for "y". However, the only uses of this I can find are in some Middle Scots passages in Appendix H II from the 14th and 15th centuries, where I'm reasonably certain it must be being used to represent a yogh (e.g. in [balȝery](https://www.dsl.ac.uk/entry/dost/bail3ery) and [ȝow](https://www.dsl.ac.uk/entry/dost/3ow_pron)).

From 1366 (p. lxxxviii):
![image](https://user-images.githubusercontent.com/6895210/158796927-ca0befb8-283a-4cfe-9087-3649b64dfa1c.png)

From 1425 (p. lxxxix):
![image](https://user-images.githubusercontent.com/6895210/158797175-f000a128-059e-4947-b46c-a9b1928d9165.png)

Would it therefore please be possible to request another cv63 variant, substituting the insular g glyph?

**Comment by psb1558** [comment]
I have some doubt whether an insular g was really used for yogh in either 1366 or 1425 (I suppose the printer had it in his case), but I have seen it used with a yogh-like distribution in  the 13th c.

So it is cv63[2].

**Reply by TheKnightWho** [reply]
Many thanks! The surrounding commentary suggests that these are transcribed directly from manuscript, which makes me think it is a (potentially dubious) stylistic choice by the Record Commission.
